core/src/main/java/org/apache/tomcat/util/net/jss/TomcatJSS.java

```diff
@@ -305,66 +320,70 @@ public void loadTomcatConfig(Document document) throws Exception {
                 "/Server/Service[@name='Catalina']/Connector[@SSLEnabled='true']",
                 document, XPathConstants.NODE);
 
-        String certDb = connector.getAttribute("certdbDir");
-        if (certDb != null)
-            setCertdbDir(certDb);
+        String certDbProp = connector.getAttribute("certdbDir");
+        if (certDbProp != null)
+            setCertdbDir(certDbProp);
 
-        String passwordClass = connector.getAttribute("passwordClass");
-        if (passwordClass != null)
-            setPasswordClass(passwordClass);
+        String passwordClassProp = connector.getAttribute("passwordClass");
+        if (passwordClassProp != null)
+            setPasswordClass(passwordClassProp);
 
-        String passwordFile = connector.getAttribute("passwordFile");
-        if (passwordFile != null)
-            setPasswordFile(passwordFile);
+        String passwordFileProp = connector.getAttribute("passwordFile");
+        if (passwordFileProp != null)
+            setPasswordFile(passwordFileProp);
 
-        String serverCertNickFile = connector.getAttribute("serverCertNickFile");
-        if (serverCertNickFile != null)
-            setServerCertNickFile(serverCertNickFile);
+        String serverCertNickFileProp = connector.getAttribute("serverCertNickFile");
+        if (serverCertNickFileProp != null)
+            setServerCertNickFile(serverCertNickFileProp);
 
-        String enableOCSP = connector.getAttribute("enableOCSP");
-        if (enableOCSP != null)
-            setEnableOCSP(Boolean.parseBoolean(enableOCSP));
+        String enableOCSPProp = connector.getAttribute("enableOCSP");
+        if (enableOCSPProp != null)
+            setEnableOCSP(Boolean.parseBoolean(enableOCSPProp));
 
-        String ocspResponderURL = connector.getAttribute("ocspResponderURL");
-        if (ocspResponderURL != null)
-            setOcspResponderURL(ocspResponderURL);
+        String ocspResponderURLProp = connector.getAttribute("ocspResponderURL");
+        if (ocspResponderURLProp != null)
+            setOcspResponderURL(ocspResponderURLProp);
 
-        String ocspResponderCertNickname = connector.getAttribute("ocspResponderCertNickname");
-        if (ocspResponderCertNickname != null)
-            setOcspResponderCertNickname(ocspResponderCertNickname);
+        String ocspResponderCertNicknameProp = connector.getAttribute("ocspResponderCertNickname");
+        if (ocspResponderCertNicknameProp != null)
+            setOcspResponderCertNickname(ocspResponderCertNicknameProp);
 
-        String ocspCacheSize = connector.getAttribute("ocspCacheSize");
-        if (StringUtils.isNotEmpty(ocspCacheSize))
-            setOcspCacheSize(Integer.parseInt(ocspCacheSize));
+        String ocspCacheSizeProp = connector.getAttribute("ocspCacheSize");
+        if (StringUtils.isNotEmpty(ocspCacheSizeProp))
+            setOcspCacheSize(Integer.parseInt(ocspCacheSizeProp));
 
```

Comment:
Unfortunately the getAttribute does not accept a default value so here it is needed the if.